packages/cli/src/Options.ts

```diff
@@ -177,11 +177,7 @@ export default class Options {
           if (arg[0] === '-') {
             throw new Error(`unexpected option: ${arg}`)
           } else {
-            if (hasGlob(arg)) {
-              config.addSourcePaths(...globSync(arg))
```

Comment:
Globbing is now centralized in `iterateSources`, so no need to do it here as well.